source-map-support.js

```diff
@@ -3,7 +3,11 @@ var path = require('path');
 
 var fs;
 try {
-  fs = require('fs');
+    fs = require('fs');
+    if (!fs.readFileSync) {
+        // If fs is empty, we are in the browser
```

Comment:
Actually, it's not empty, it's: https://www.npmjs.com/package/browserify-fs

How about:

```js
try {
  fs = require('fs');
  if (!fs.existsSync || !fs.readFileSync) {
    // fs doesn't have all methods we need
    fs = null;
  }
} catch (err) {
  /* noop */
}
```

also, please keep the indentation at 2 spaces :)